swagger.yaml

```diff
@@ -4286,10 +4286,6 @@ definitions:
         description: "The desired network type used."
         enum: ['OpenShiftSDN', 'OVNKubernetes']
         default: 'OpenShiftSDN'
-      schedulable_masters:
```

Comment:
Yes, this is because we don't allow the user to change the scheduling of masters when the number of hosts in the cluster is less than 5. At cluster creation time the number of hosts are 0 and the mandatory value for the `schedulable_masters` property is `true`

Reply:
Also I don't see anything in the code base that indicates that we can't set `schedulable_masters` to any value at cluster creation time.

I don't see any reason to remove this.